spec/source_finder/source_file_globber_spec.rb

```diff
@@ -84,12 +85,12 @@
         source_file_globber.extra_ruby_files_arr = ['mumble.rb']
         source_file_globber.extra_source_files_arr =
           ['AUTOEXEC.BAT', 'CONFIG.SYS'] +
-          source_file_globber.extra_ruby_files_arr
+            source_file_globber.extra_ruby_files_arr
         source_file_globber.ruby_file_extensions_arr =
           ['.RB', '.RAKE']
         source_file_globber.source_file_extensions_arr =
           source_file_globber.ruby_file_extensions_arr +
-          ['JAVA', 'C#']
+            ['JAVA', 'C#']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.